RNWCPP/ReactUWP/Views/ViewViewManager.cpp

```diff
@@ -200,34 +223,27 @@ XamlView ViewViewManager::CreateViewPanel(int64_t tag)
 
 XamlView ViewViewManager::CreateViewControl(int64_t tag)
 {
-  // Create the ViewPanel which will be nested under the ContentControl
-  XamlView newViewPanelXamlView(ViewPanel::Create().as<XamlView>());
-  auto panel = newViewPanelXamlView.as<ViewPanel>();
-  panel->VerticalAlignment(winrt::VerticalAlignment::Top);
-
-  // Create the ContentControl as the outer/containing element, nest the ViewPanel, set default properties
+  // Create the ViewControl as the outer/containing element, nest the ViewPanel, set default properties
   auto contentControlPtr = ViewControl::Create();
-  auto contentControl = contentControlPtr.as<winrt::ContentControl>();
-  contentControl.Content(newViewPanelXamlView);
 
   // TODO: Remove once existing clients stop using TouchableNativeFeedback.uwp
-  contentControl.Tapped([=](auto &&, auto &&args)
+  contentControlPtr->Tapped([=](auto &&, auto &&args)
   {
     DispatchEvent(tag, "topPress", std::move(folly::dynamic::object("target", tag)));
     args.Handled(true);
   });
```

Comment:
we should be able to remove this now, LPC and outlook have moved off the old touchable

Reply:
I'll create a separate pull request for that then.